### ISSUE TITLE
Create E2E tests 

### ISSUE THREAD

**Issue by llowinge** [citrusframework/yaks#206]
Would be really great if this framework would contain E2E tests which could verify basic yaks workflow as it would create a contract between users and their expectations.

**Scenario 1** 

1. We have yaks-config.yaml containing temporary namespace settings.
2. Create namespace yaks-global-operator
3. yaks install (globally)
4. yaks test (verify temporary namespace was created; no new operator was created in temp namespace and tests passed)

**Scenario 2** 

1. We have yaks-config.yaml containing temporary namespace settings.
2. Create namespace yaks-non-global-operator
3. yaks install (non-globally) - maybe this step is not needed ?
4. yaks test (verify temporary namespace was created and tests passed)

@christophd FYI

**Comment by christophd** [comment]
Yes, and I think the documentation should outline these workflows and list its individual benefits and possible downsides
